OneCommunityGlobal/HighestGoodNetworkApp
# Nida - Fix making a user inactive and then editing their blue squares bug

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/147a66d9-6fc5-4dac-a574-0d3aa12b83bc)


# Fixes (bug list priority low)
- Removed duplicate `useEffect` hooks that was getting invoked by the exact same function calls and had the same dependency.

## How to test:
1. check into current branch
2. log as admin user
3. go to User Profile (make them inactive by clicking on the green dot) → Delete a blue square
4. verify that the blue square gets deleted without any pop up alert appearing on the screen

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/32549172/5ab3742b-f812-4ed8-987a-14aae75d5762
